src/test/java/seedu/address/logic/parser/AddressBookParserTest.java

```diff
@@ -79,8 +83,8 @@ public void parseCommand_exit() throws Exception {
     public void parseCommand_find() throws Exception {
         List<String> keywords = Arrays.asList("foo", "bar", "baz");
         FindCommand command = (FindCommand) parser.parseCommand(
-                FindCommand.COMMAND_WORD + " " + keywords.stream().collect(Collectors.joining(" ")));
-        assertEquals(new FindCommand(new NameContainsKeywordsPredicate(keywords)), command);
+                FindCommand.COMMAND_WORD + " " + PERSON_ENTITY_STRING + " " + "foo bar baz");
```

Comment:
Perhaps can replace `PERSON_ENTITY_STRING` with `ParserUtil.PERSON_ENTITY_STRING` to fix the coverage failure